values.yaml

```diff
@@ -64,7 +64,7 @@ global:
       nodes: 6
 
   # hosting: [local | aws | metal] the target environment to deploy to
-  hosting: local
+  hosting: aws
 
```

Comment:
What do we want the default to be?
At the moment, a `helm install palisade .` works locally, with this we would have to be slightly more explicit in what is probably the more-common use-case at the moment (and in general, any aws deploys are probably automated so whether one is the default or not doesn't effect time typing commands)

Reply:
I'll switch that back to local, forgot it.